doc/guides/writing-tests.md

```diff
@@ -231,9 +231,9 @@ assert.throws(
 For performance considerations, we only use a selected subset of ES.Next
 features in JavaScript code in the `lib` directory. However, when writing
 tests, for the ease of backporting, it is encouraged to use those ES.Next
-features that can be used directly without a flag in [all maintained branches]
-(https://github.com/nodejs/lts). [node.green](http://node.green/) lists
-available features in each release.
+features that can be used directly without a flag in
+[all maintained branches](https://github.com/nodejs/lts).
```

Comment:
There are 5 links in the doc without references. Should I move them all to the bottom?